condor.go

```diff
@@ -332,6 +328,8 @@ func (cl *CondorLauncher) stopHandler(condorPath, condorConfig string) func(d am
 			}
 			log.Infof("Output of 'condor_rm %s':\n%s", condorID, condorRMOutput)
 		}
+
+		d.Ack(false)
```

Comment:
One potential issue I didn't think of -- what do we want to do when errors happen, as far as acknowledgement?

The options I see are:

1. Always acknowledge and never requeue, even if there's an error. (in this case, put the `d.Ack` back where it was, but add `defer` so it happens when the function exits)
2. Reject with a requeue in case of error (where something above would `return`, also add a `d.Reject(true)`, or add some sort of wrapper which catches errors and does the ack)
3. Option 2, but conditional -- what we do in dewey (and maybe also info-typer) is requeue only if the original delivery wasn't itself a redelivery, i.e. retry once.

Probably one of these needs to be chosen; I think without that, if it's hitting errors it'll deliver the first batch and then just hang -- the server will expect acks or rejects, so it won't send more messages, but the service will think it's done with everything.

Reply:
I think I like option 3, especially since we can now have so many jobs running that may need to be stopped. That will give us a 2nd chance to cleanup multiple jobs in case of a temporary problem, but if it's something more persistent the messages won't get backed up.